draft-ietf-scitt-architecture.md

```diff
@@ -158,59 +160,59 @@ Statement:
 In COSE, Feed is a dedicated header attribute in the protected header of the Envelope.
 
 Registry:
 
-: the verifiable append-only data structure that stores Claims in a Transparency Service often referred to by the synonym log or ledger. SCITT supports multiple Registry and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees.
+: the verifiable append-only data structure that stores statements in a Transparency Service often referred to by the synonym log or ledger. SCITT supports multiple Registry and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees.
 
 Transparency Service (TS):
 
 : an entity that maintains and extends the Registry, and endorses its state. A Transparency Service is often referred to by its synonym Notary. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry. The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
 
 Receipt:
 
-: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Registry. It consists of a Registry-specific inclusion proof, a signature by the Transparency Service of the state of the Registry, and additional metadata (contained in the countersignature protected headers) to assist in auditing.
+: a Receipt is a special form of COSE countersignature for Signed Statments that embeds cryptographic evidence that the Signed Statement is recorded in the Registry. A Receipt consists of a Registry-specific inclusion proof, a signature by the Transparency Service of the state of the Registry, and additional metadata (contained in the countersignature's protected headers) to assist in auditing.
 
 Registration:
 
-: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Registry, producing a Receipt, and returning it to the submitter.
+: the process of submitting a Signed Statement to a Transparency Service, applying the Transparency Service's registration policy, storing it in the Registry, producing a Receipt, and returning it to the submitting Issuer.
 
 Registration Policy:
 
-: the pre-condition enforced by the TS before registering a Claim,
-based on its Envelope (notably the identity of its Issuer)
-and on prior claims already in the Registry.
+: the pre-condition enforced by the Transparency Service before registering a Signed Statement,
+based on metadata contained in its COSE Envelope (notably the identity of its Issuer)
+and on prior Signed Statements already added to a Registry.
 
 Transparent Claim:
```

Comment:
```suggestion
Transparent Statement:
```

Reply:
We have to check consistent use of Signed Statement vs. Transparent Statement.